jf_agent/jf_jira/jira_download.py

```diff
@@ -123,52 +123,16 @@ def detect_and_repair_custom_fields(
 
     kwargs = dict()
 
-    logger.info("Beginning detection of custom field mismatches")
+    logger.info(f'Beginning detection of custom field mismatches. Submit issues for redownload: {submit_issues_for_repair}')
```

Comment:
Nit: could you run `pdm run black jf_agent/jf_jira/jira_download.py` to reformat a few long lines of code in this change?